### ISSUE TITLE
Error when redirecting from return URL

### ISSUE THREAD

**Issue by flizana** [cornershop/python-tbk#7]
I have successfully implemented WebpayPlus with the help of this repo. Although when I tell the server to redirect to the URL provided by Transbank (with `token_ws=blabla` as a query parameter) after `get_transaction_result` call, the following page is displayed:

![screen shot 2019-02-27 at 21 24 34](https://user-images.githubusercontent.com/6475823/53532614-4796fa80-3ad6-11e9-8d95-8a4cb2ec684b.png)

BTW, I am already calling `acknowledge_transaction` before telling the server to redirect. The odd thing is if I click on the link it leads me to my final URL.

**Comment by nanocatdemen** [comment]
I have exactly the same problem, did you find the cause?

**Comment by israelgp24** [comment]
> I have successfully implemented WebpayPlus with the help of this repo. Although when I tell the server to redirect to the URL provided by Transbank (with `token_ws=blabla` as a query parameter) after `get_transaction_result` call, the following page is displayed:
> 
> ![screen shot 2019-02-27 at 21 24 34](https://user-images.githubusercontent.com/6475823/53532614-4796fa80-3ad6-11e9-8d95-8a4cb2ec684b.png)
> 
> BTW, I am already calling `acknowledge_transaction` before telling the server to redirect. The odd thing is if I click on the link it leads me to my final URL.
 Did you figure it out ?

**Comment by israelgp24** [comment]
Hi @flizana  ,  I'am getting the same problem
Did you figure it out ?

**Comment by flizana** [comment]
@israelgp24  Even though this page is displayed, the transaction is approved (by checking the authorization code). I think it must be a bug inside Webpay (Transbank). I'm still looking for the solution.

**Comment by nanocatdemen** [comment]
I had the same problem. It turns out that the form element didn't have method="post", so it was a GET request, not a POST request

**Comment by nanocatdemen** [comment]
Note: My "redirect" it just a form with a hidden token_ws element, submitted programmatically with js

**Comment by israelgp24** [comment]
@flizana , really ? do you get a transaction approved ?, i used inittransaction  and i got a token and a url , but when i made gettransactionresult using that token i got a timeout error

**Comment by israelgp24** [comment]
@nanocatdemen 
when you say form you mean the webpay form ?

**Comment by flizana** [comment]
> Note: My "redirect" it just a form with a hidden token_ws element, submitted programmatically with js

I know, my redirect is the same, a form with hidden `token_ws` input element. @nanocatdemen  did you solve this problem?

**Comment by nanocatdemen** [comment]
Here I'm using django. The "success" view

```
result = webpay.get_transaction_result(request.POST['token_ws'])
redirect_url = result['urlRedirection']
webpay.acknowledge_transaction(request.POST['token_ws'])
```

**Comment by nanocatdemen** [comment]
Using django. The "success" view

```
result = webpay.get_transaction_result(request.POST['token_ws'])
redirect_url = result['urlRedirection']
webpay.acknowledge_transaction(request.POST['token_ws'])
...
```

Renders this html template
```
<form method="POST" action="{{ redirect_url }}" >
  <input type="hidden" name="token_ws" value="{{ token_ws }}" />
</form>
<script>
setTimeout(function(){
    document.getElementsByTagName('form')[0].submit();
}, 3000);
</script>
```

**Comment by pedroburon** [comment]
this issues should be resolved with documentation #3
